index.html

```diff
@@ -55,7 +55,7 @@
             >
           </li>
         </ul>
-        <a href="javascript:void(0);" class="icon" onclick="toggleMobileNav()">
+        <a id="hamburger" href="javascript:void(0);" class="icon" onclick="toggleMobileNav()">
```

Comment:
Try to make sure that whenever any link in the menu is clicked, the sidebar collapses.
Currently after clicking the `हिन्दी में देखिये` link or `Read in English` link, the sidebar doesn't collapse.

![image](https://github.com/rbjks/rbjks.github.io/assets/43912470/cb5fa217-ddd0-45cd-baac-997a2c500123)